README.md

```diff
@@ -492,3 +492,10 @@ I think it should be here.
 *first-time on the kottans pre-course:*
 
 ![first-time on the kottans pre-course](gif/first-time-kottans.gif)
+
+
+
+
```

Comment:
you don't need all this empty space, markdown removes in automatically on display
one empty line is sufficient :slightly_smiling_face: